sql/report_queries/acrl/collections_expenditures/ACRL Collection Expenditures B.sql

```diff
@@ -0,0 +1,189 @@
+--This query is for materials purchased, not other cost ACRL suggest may be included under "All other materials/services cost". 
+--This query does not include transactions that are made at the invoice level that are not prorated. 
+--If the total of invoice transactions not distributed to the Invoice Lines is needed (e.g. adjustmetns), then the ACRL Expenditures C should be run jointly. 
+--An export to Excel will provide the total amount paid by summing the 'transaction_amount_per_invl_converted' column
```

Comment:
Just similar comments to query A. Now I understand this. this comment. It sort of through for a loop the first time around. I think I was just expected the sum to be in the report. Also, like with A, there are some formatting issues and the missing ";".

Reply:
Yes, a big part of it is the same. :)